### PR TITLE
add japansese translations.

### PR DESCRIPTION
**Pull request by yositani2002**

### CODE REVIEW

File: Resources/translations/SonataClassificationBundle.ja.xliff [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            <trans-unit id="classification.list_mode">
+                <source>classification.list_mode</source>
+                <target>リスト</target>
+            </trans-unit>
+            <trans-unit id="classification.tree_mode">
+                <source>classification.tree_mode</source>
+                <target>ツリー</target>
+            </trans-unit>
+
+        </body>
```
Comment (core23) [inline Resources/translations/SonataClassificationBundle.ja.xliff]: Please remove the empty line

**Comment by core23** [comment]
Ping @yositani2002

**Comment by OskarStark** [comment]
@core23 if he didn't answer, lets merge it as it is

**Comment by yositani2002** [comment]
removed empty line.

**Comment by core23** [comment]
Thank you @yositani2002

**Comment by OskarStark** [comment]
thank you @yositani2002

**Comment by OskarStark** [comment]
@yositani2002 would you like to check the other bundles too?

**Pull request merged**